### ISSUE TITLE
Fix DAP issues for Banner

### ISSUE THREAD

**Issue by youda97** [IBM/carbon-components-angular#107]
## Basic

Level | Description
-- | --
V | Each form control must have associated label. | Details

## Dynamic toast

Level | Description
-- | --
V | Each form control must have associated label. | Details
V | All content must reside within a WAI-ARIA landmark or labelled region role.

## Dynamic

Level | Description
-- | --
V | Each form control must have associated label. | Details
V | All content must reside within a WAI-ARIA landmark or labelled region role.

## Toast

Level | Description
-- | --
V | Each form control must have associated label.

**Comment by cal-smith** [comment]
# Dynamic toast
2.  false positive, since the preview page doesn't have a `main` tag. Unfortunately there's no reasonable way for us to add one at the moment. Apps should of course render into a `main` or other valid region.

# Dynamic
2. false positive, since the preview page doesn't have a `main` tag. Unfortunately there's no reasonable way for us to add one at the moment. Apps should of course render into a `main` or other valid region.

**Comment by youda97** [comment]
closed by #156
